packages/components/dropdown/src/dropdown.vue

```diff
@@ -152,6 +158,39 @@ export default defineComponent({
       return props.id || defaultTriggerId
     })
 
+    // The goal of this code is to focus on the tooltip triggering element when it is hovered.
+    // This is a temporary fix for where closing the dropdown through pointerleave event focuses on a
+    // completely different element. For a permanent solution, remove all calls to any "element.focus()"
+    // that are triggered through pointer enter/leave events.
+    watch(
+      [triggeringElementRef, toRef(props, 'trigger')],
+      ([triggeringElement, trigger], [prevTriggeringElement]) => {
+        const triggerArray = isArray(trigger) ? trigger : [trigger]
+        if (prevTriggeringElement?.$el?.removeEventListener) {
+          prevTriggeringElement.$el.removeEventListener(
+            'pointerenter',
+            onAutofocusTriggerEnter
+          )
+        }
+        if (triggeringElement?.$el?.removeEventListener) {
+          triggeringElement.$el.removeEventListener(
+            'pointerenter',
+            onAutofocusTriggerEnter
+          )
+        }
+        if (
+          triggeringElement?.$el?.addEventListener &&
+          triggerArray.includes('hover')
+        ) {
+          triggeringElement.$el.addEventListener(
+            'pointerenter',
+            onAutofocusTriggerEnter
```

Comment:
Do we need to remove the onAutofocusTriggerEnter function in onBeforeUnmount?

Reply:
Is there a scenario where dropdown is unmounted and its slot content isn't discarded?

Reply:
I added it in case the virtual dom diffing keeps that node around. Better safe than sorry I suppose.